content/en/blog/_posts/2025-xx-xx-jobs-managedby-goes-ga.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: blog
+title: "Kubernetes v1.35: Job Managed By Goes GA"
+date: 2025-XX-XX
+draft: true
+slug: kubernetes-v1-35-job-managedby-for-jobs-goes-ga
+author: >
+  [Dejan Zele Pejchev](https://github.com/dejanzele) (G-Research),
+  [Michał Woźniak](https://github.com/mimowo) (Google)
+---
+
+In Kubernetes v1.35, the `managedBy` field for Jobs has reached general availability (GA).  
+This post gives a tour of what `managedBy` does, why it was added, and how it is used today, most notably by
+[MultiKueue](https://kueue.sigs.k8s.io/docs/concepts/multikueue/) for multi-cluster Job dispatching.
+
+## Why delegate Job reconciliation?
+
+Before `managedBy`, every `batch/v1` Job in a cluster was completely “owned” by the built-in Job controller.
+
+That behaviour works well for single-cluster workloads, but it becomes limiting once you introduce multi-cluster
+schedulers and various controllers on top of Jobs.
+
+A few examples:
+- **Multi-cluster orchestration.**  
+  In MultiKueue’s manager-worker architecture, a Job in the management cluster needs its status mirrored from a Job
+  running in a worker cluster. For that to work reliably, we must be able to tell the built-in Job controller to
+  _not reconcile_ certain Jobs.
+- **Custom Job controllers.**  
+  Projects like JobSet, Kubeflow Training Operator, KubeRay, Tekton Pipelines, and AppWrapper provide their own
+  reconciliation logic around Job-like resources and integrate with Kueue and MultiKueue. They need a standard way to
+  say “this controller is responsible for status” without fighting the Job controller.
+- **Operational safety.**  
+  Disabling the Job controller cluster-wide using `--controllers=-job,*` is heavy-handed, often not possible on managed
+  control planes, and doesn’t work well when the same cluster plays both manager *and* worker roles.
+
+The `managedBy` field solves this by letting you delegate Job synchronization on a **per-Job** basis instead of flipping
+a global switch.
+
+## How `spec.managedBy` works
+
+`spec.managedBy` indicates which controller manages this Job.
+
+The semantics are intentionally simple:
+- If `spec.managedBy` is **unset**, or set to the reserved value `kubernetes.io/job-controller`,  
+  the built-in Job controller reconciles the Job as usual.
+- If `spec.managedBy` is set to **any other valid value**, the built-in Job controller **Skips reconciliation** for that Job, 
+  and continues to reconcile all other Jobs in the cluster.
```

Comment:
This feels a bit redundant wrt `spec.managedBy`. Also it would be nice to mention that the field is immutable (and why).

Maybe:

The spec.managedBy field indicates which controller is responsible for the Job, specifically there are two modes of operation:
- **Standard**: if unset or set to the reserved value `kubernetes.io/job-controller`, the built-in Job controller reconciles the Job as usual (standard behavior).
- **Delegation**: If set to any other value, the built-in Job controller skips reconciliation entirely for that Job.

To prevent orphaned Pods or resource leaks, this field is immutable. You cannot transfer a running Job from one controller to another.